src/conditions/mod.rs

```diff
@@ -34,3 +37,66 @@ dyn_clone::clone_trait_object!(ConditionConfig);
 pub type ConditionDescription = ComponentDescription<Box<dyn ConditionConfig>>;
 
 inventory::collect!(ConditionDescription);
+
+#[derive(Debug, Deserialize, Serialize, Clone)]
+#[serde(untagged)]
+pub enum AnyCondition {
+    String(String),
+    Map(Box<dyn ConditionConfig>),
+}
+
+impl AnyCondition {
+    pub fn build(&self) -> crate::Result<Box<dyn Condition>> {
+        match self {
+            AnyCondition::String(s) => RemapConfig { source: s.clone() }.build(),
+            AnyCondition::Map(m) => m.build(),
+        }
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[derive(Deserialize, Debug)]
+    struct Test {
+        condition: AnyCondition,
+    }
+
+    #[test]
+    fn deserialize_anycondition_default() {
+        let conf: Test = toml::from_str(r#"condition = ".nork == false""#).unwrap();
+        assert_eq!(
+            "Test { condition: String(\".nork == false\") }",
+            format!("{:?}", conf)
+        )
+    }
+
+    #[test]
+    fn deserialize_anycondition_check_fields() {
+        let conf: Test = toml::from_str(
+            r#"condition.type = "check_fields"
+               condition."norg.equals" = "nork""#,
+        )
```

Comment:
This would be a great case for [indoc](https://github.com/dtolnay/indoc):

```suggestion
        let conf: Test = toml::from_str(indoc! {
            condition.type = "check_fields"
            condition."norg.equals" = "nork"
        })
```

Reply:
This is really nice! I didn't know about indoc. There are lots of places where we can use this!!